### DISCUSSION TITLE
Print progress time remaining, elapsed

### DISCUSSION THREAD

**Discussion by radarOverhead** [Desuuuu/klipper#141] [General]
Thanks for putting together this software for us ..  Its great !

One small issue on my install on a cr10s Pro V2.  When issuing a M73, the percentage bar updates fine but the time elapsed is a a long decimal number like 0.0074074074.  During a print, this number changes constantly - even if a M73 was not issued.  Is it something that I'm doing wrong.  

My M73 commands look like  M73 p13 r45

Everything else looks and performs great!

Thanks in advance

**Comment by Desuuuu** [comment]
Are you running Klipper with Python 3?
